gradle.properties

```diff
@@ -37,7 +37,6 @@ kotlin.code.style=official
 
 # Disable build features that are enabled by default,
 # https://developer.android.com/build/releases/gradle-plugin#default-changes
-android.defaults.buildfeatures.resvalues=false
```

Comment:
IMO re-enabling `resValues` for that is not good. We already have specific sourceSet for all flavors.
Also, I'm not sure that Prod variant is actually expected to be installed by regular devs, since it needs production keys etc.
This makes the actual Demo app name appearance worse.

Reply:
@SimonMarquis 
It seems hard it to do with resValue from the error logs. I can do same thing with another way without resValues.
Before refactor that I also wonder `prod` usage, this project just needs the `prodRelease`?